suites/squid/cephfs/tier-3_cephfs_bugs.yaml

```diff
@@ -252,3 +254,16 @@ tests:
       polarion-id: CEPH-83591136
       desc: Verify Ceph Health Warning for Slow MDS Requests After Cluster Installation
       abort-on-fail: false
+  - test: # https://bugzilla.redhat.com/show_bug.cgi?id=2294478
+      name: Prevent scrub running using standby MDS
+      module: cephfs_bugs.prevent_scrub_with_standby_mds.py
+      polarion-id: CEPH-83592512
+      desc: Prevent scrub running using standby MDS
+      abort-on-fail: false
```

Comment:
@julpark-rh here you can add another field "comments: Active bug - 2294478" so that it reflects in the comments section of our CI email reports. Merging for now, please note and make necessary changes in the xfstests PR.